src/components/utility/UtilitySrv.js

```diff
@@ -61,7 +61,15 @@
                     return prom;
                 };
             };
-            
+
+            UtilitySrv.fn.isValidNumber = function(number){
```

Comment:
isValidNumber is not a function which related to function (singletonPromise function location is bad as well)